src/components/accessories/laboratory/Exams.tsx

```diff
@@ -54,11 +54,20 @@ export const Exams: FC = () => {
     (state: IState) => state.laboratories
   );
 
+  useEffect(() => {
+    setFilter({ ...filter, page: page });
+  }, [page]);
```

Comment:
Not including `filter` in the dependencies array could cause some consistency problems. A correct implementation of this effect function should be like the following:
```suggestion
  useEffect(() => {
    setFilter((prevState) => ({ ...prevState, page: page }));
  }, [page]);
```